content/accessibility/guidelines.mdx

```diff
@@ -0,0 +1,186 @@
+
+<DoDontContainer stacked>
+    <Do>
+       <img src="https://user-images.githubusercontent.com/40274682/102424769-bc476b80-3fc0-11eb-9831-c229ada406c0.png" alt="" />
+       <Caption>Buttons with sufficient text contrast</Caption>
+    </Do>
+    <Dont>
+       <img src="https://user-images.githubusercontent.com/40274682/102424747-afc31300-3fc0-11eb-9e98-ecd806f6e266.png" alt="" />
+       <Caption>Buttons with sufficient text contrast</Caption>
+    </Dont>
+</DoDontContainer>
+
+**Normal text (14pt)**
+4.5:1 minimum
+
+**Large text** (14pt bold + , 18pt +)**
+3:1 minimum
+
+**Graphical objects and user interface components**
+3:1 minimum
+
+**Note:** Disabled controls do not need to hit any specific contrast levels.
+
+**Official WCAG guidelines**
+- [1.4.3 - Contrast (Minimum)(Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)
+- [1.4.11 - Non-text Contrast (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html)
+- [1.4.1 - Use of color (Level AA)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html)
+
+#### Use of color
+Adding in color brings delight to user experiences. The key point in [1.4.1 Use of Color](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html) is to make sure that it's not the sole identifier. If you have an important transition that is only a change in color, make sure that color change has a 3:1 contrast ratio. 
+
+A great way to test for this is to navigate your designed experience in greyscale mode. 
+
+<DoDontContainer stacked>
+    <Do>
+       <img src="https://user-images.githubusercontent.com/40274682/102438332-eb65d900-3fd0-11eb-8238-0e826f53b939.png" alt="" />
+       <Caption>Using color and a different shape to indicate issue status</Caption>
+    </Do>
+    <Dont>
+       <img src="https://user-images.githubusercontent.com/40274682/102438275-d0936480-3fd0-11eb-8b1c-94bbb0033c1f.png" alt="" />
+       <Caption>Using only color to indicate issue status</Caption>
+    </Dont>
+</DoDontContainer>
+
+### Typography
+#### Headings
+Properly labeled headings are one of the most important things a designer can make sure to do for developer handoff. [68% of 1207 surveyed screen reader users](https://webaim.org/projects/screenreadersurvey8/) responded that they are first likely to navigate via the headings when trying to find information on a lengthy web page.
+
+Use the semantic structure for headings whenever possible on paragraphs and sections (`h1`-`h6`). 
+
+There should only be one `h1` per web page. (Markdown on a page is an exception)
+
+**Note:** Headings on a page don't need to visually match the semantic hierarchy, but should take on the appropriate role. 
```

Comment:
The last part means that coded headings are sometimes avoided because the style doesn't fit. Think about the file names in the Files Changed tab. They should be H3s, but aren't because the styling doesn't fit. They would still need to be coded as an H3, but apply different visual styling.

Reply:
I made a suggestion to rephrase.